ingestion_server/ingestion_server/cleanup.py

```diff
@@ -273,25 +284,68 @@ def _clean_data_worker(rows, temp_table, sources_config, all_fields: list[str]):
 
 
 def save_cleaned_data(result: dict) -> dict[str, int]:
-    log.info("Saving cleaned data...")
     start_time = time.perf_counter()
 
     cleanup_counts = {field: len(items) for field, items in result.items()}
     for field, cleaned_items in result.items():
         # Skip the tag field because the file is too large and fills up the disk
-        if field == "tag":
+        if field == "tag" or not cleaned_items:
             continue
-        if cleaned_items:
-            with open(f"{field}.tsv", "a") as f:
-                csv_writer = csv.writer(f, delimiter="\t")
-                csv_writer.writerows(cleaned_items)
+
+        with open(TMP_DIR.joinpath(f"{field}.tsv"), "a", encoding="utf-8") as f:
```

Comment:
Kind of just a nit, but I usually see usage of the `/` operator to join path segments, rather than `joinpath`, although we seem to have a mix of both in our project (maybe more `/` versions, though?). Anyway, just a nit/nothing.

```suggestion
        with open(TMP_DIR / f"{field}.tsv", "a", encoding="utf-8") as f:
```

Reply:
I took the idea from your recent changes in managing the files for waveform. I like how explicit it is in the `with` line. 

https://github.com/WordPress/openverse/blob/d562f4719aefdf7d90c13ca6738a5e8c27ce352f/api/api/utils/waveform.py#L79

I found the form with ` / ` handy for longer paths in a separate line.